src/components/project/dto/project.dto.ts

```diff
@@ -131,6 +131,7 @@ class Project extends Interfaces {
 
   readonly marketingRegionOverride: Secured<IdOf<Location> | null>;
   readonly fieldRegion: Secured<ID | null>;
+  readonly marketingRegion: Secured<IdOf<Location> | null>;
```

Comment:
```suggestion
  @Calculated()
  readonly marketingRegion: Secured<IdOf<Location> | null>;
```
This let's the permission system know that edits are not applicable.

Also seen here:
https://github.com/SeedCompany/cord-api-v3/blob/e867d3f50f37fb18c670aef0cd01462b024496c4/src/components/engagement/dto/engagement.dto.ts#L96-L106